.github/workflows/rust.yml

```diff
@@ -40,6 +40,14 @@ jobs:
     - run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
     - run: wasm-pack test --chrome --headless
 
+  wasmbench:
+    name: Wasm Bench
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```

Comment:
Does it go somewhere?

Reply:
Thanks for the reminder, now uploaded as an artifact (look on https://github.com/rocicorp/repc/pull/56/checks) like wasm-size.txt.